src/game.js

```diff
@@ -358,10 +357,7 @@ load(
 			map.render();
 			mapText.render();
 			player.render();
-			enemies.forEach(function(enemy){
```

Comment:
why the change here?

Reply:
Consistency with the line below it. Not really needed but I snuck it in